data/EW/metadata.json

```
@@ -1,5 +1,70 @@
 {
-  "contributors": [],
+  "contributors": [
+    {
+      "Type": "CDR Company",
+      "Name": "Eion",
+      "Affiliation": "",
+      "initial": "1.0",
+      "notes": "",
+      "enhanced-weathering": "1.0, 1.1"
```

Would it be possible to make a couple of tweaks here?

- consistently use lowercase keys
- roll up `initial` and `enhanced-weathering` into a single `versions` array

```suggestion
      "type": "CDR Company",
      "name": "Eion",
      "affiliation": "",
      "versions": ["1.0", "1.1"],
      "notes": "",
```